examples/aws_lambda/lazy_aws_lambda_config.yaml

```diff
@@ -4,9 +4,7 @@ function_name: bolt_py_function
 handler: lazy_aws_lambda.handler
 description: My first lambda function
 runtime: python3.8
-# role: lambda_basic_execution
-# Have lambda:InvokeFunction & lambda:GetFunction in the allowed actions
```

Comment:
The required roles are noted in the [`README.md`](https://github.com/slackapi/bolt-python/tree/main/examples/aws_lambda#lazy-lambda-listener-example-bolt-app), so these comments were removed to avoid confusion! 📚

Reply:
yeah, makes sense!